docs/dev/communication.md

```diff
@@ -0,0 +1,15 @@
+# Communication Guidelines
+
+pyDARN is under the Data Analysis Working Group that has setup some guidelines on [communication](https://superdarn.github.io/dawg/documents/communication-guidelines/) to encourage positive collaborative environment.
+Please follow these guidelines when commenting on GitHub. 
+
+Also please read workflow documentation on issues and [pull requests](dev/PR.md) to communicate properly for collaborative work.
+
+## Disputes Procedure
+
+If one of the members have a dispute that violates the communication guidelines and is creating a negative environment in the pyDARN community, please follow the procedure for resolving the conflict:
```

Comment:
```suggestion
If members have a dispute that violates the communication guidelines, and creates a negative environment in the pyDARN community, please follow this procedure to resolve the conflict:
```
Same tense for the whole sentence, but equally could be the opposite tense, I've gone with future.